src/commands/activePullRequests.ts

```diff
@@ -12,6 +12,14 @@ export const ActivePullRequests = (
   const codeReviews = Store.get(context)
   if (!codeReviews) return
 
+  if (
+    !codeReviews?.pendingUserCodeReviews?.length &&
+    !codeReviews?.userAuthoredCodeReviews?.length
+  ) {
+    window.showInformationMessage('Pullflow: You have no PRs waiting for you.')
```

Comment:
Are we not able to use icons?

Reply:
@srzainab we can't use icons in pops in vscode